src/watermelonSidebar.ts

```diff
@@ -126,6 +126,11 @@ export default class WatermelonSidebar implements vscode.WebviewViewProvider {
           });
           break;
         }
+        case "link": {
+          this.reporter?.sendTelemetryEvent(data.source, { link: data.link });
```

Comment:
Why is the event name programmatically retreived from data.source, instead of naming it "dailySummary"?

Not only this is confusing to the person reading the code, but it makes the event name get stored as undefined. 

<img width="383" alt="Screen Shot 2022-07-13 at 5 39 43 PM" src="https://user-images.githubusercontent.com/8325094/178849057-f46454d7-db36-4825-a195-2c417c517919.png">

Reply:
Ah, I see the error, it comes in `data.data.source`.
Makes #302 way more important.
It was fixed to allow the extensibility of the case.

Reply:
Also, this function lives in the handler for the webview, so it allows _any_ link to be opened externally and tracked. 
We should do so for the ones in the dropdown menu, it allows us to funnel to see if the airtable form is too much for people to fill.

Reply:
Also, and I' severely ahead of myself, we could track with this function ToDos links in the comments.